examples/cloud-discovery/hazelcast_cloud_discovery_example.py

```diff
@@ -2,16 +2,16 @@
 
 client = hazelcast.HazelcastClient(
     # Set up cluster name for authentication
-    cluster_name="YOUR_CLUSTER_NAME",
+    cluster_name="vprod",
```

Comment:
do we really want to provide these?

Reply:
Oops:
https://github.com/hazelcast/hazelcast-python-client/pull/676/commits/30c6b419da86cac240e75683d306654c4713a1c2